.github/workflows/cargo-semver-check.yml

```diff
@@ -14,15 +14,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - name: Semver check crates without "unstable" feature
+      - name: Semver check host crates
         uses: obi1kenobi/cargo-semver-checks-action@v2
         with:
           feature-group: default-features
-      - name: Semver check crates with "unstable" feature
```

Comment:
Maybe we should leave this in, just in case we add an unstable feature in the future? Not sure. Feel free to argue against that.

Reply:
I would add such a check when we add such a crate. There could be any other feature as well.